### DISCUSSION TITLE
在routeros的container里运行的问题

### DISCUSSION THREAD

**Discussion by csy59** [v2rayA/v2rayA#980] [Q&A]
我在routeros的container里运行v2raya的docker版本，点击启动的时候提示failed to start v2ray-core: ExecCommands: iptables -w 2 -N DROP_SPOOFING iptables v1.6.0: can't initialize iptables table `filter': Module is wrong version Perhaps iptables or your kernel needs to be upgraded. : exit status 3
这个错误是什么引起的，应该怎么解决啊

**Comment by hccnm** [comment]
请问你后面是怎么解决的？

**Comment by InvisibleRain** [comment]
RouterOS is not officially supported by v2rayA, however you can use [tun2socks](https://github.com/xjasonlyu/tun2socks) to route traffic into SOCKS proxy provided by v2rayA or any other proxy client.
See: https://forum.mikrotik.com/viewtopic.php?t=203410&sid=8f0f6594d6a173bc1ee7595ba4ad839b#p1080756
